### PR TITLE
i18n support for Bosnian language

### LINKED ISSUE #1860: Feature: i18n support - Bosnian
## Why?

Recently novu introduced an easy way to add translations for the @novu/notification-center component interface. So we can either specify a translation object or pass a premade translation.

**Note:**

Bosnian, though similar to Croatian and other Balkan languages, is widely used (almost by 50% population) in Bosnia.

## How?

- Add a new translation file here: packages/notification-center/src/i18n/languages. You can follow the en.ts example.

- Add your translation import to the TRANSLATIONS object at packages/notification-center/src/i18n/lang.ts

- Add the language code to I18NLanguage type at lang.ts file.

I'd like to work on this issue.

### PR DESCRIPTION
**Pull request by abhilipsasahoo03**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Added i18n support for Bosnian language
- **Why was this change needed?** (You can also link to an open issue here)
Fixes #1860

### CODE REVIEW

**Comment by abhilipsasahoo03** [comment]
@davidsoderberg @scopsy kindly review!

**Review by p-fernandez — CHANGES_REQUESTED** [review]
Please update your branch with the latest changes we added in `main`. Now the file https://github.com/novuhq/novu/blob/main/packages/notification-center/src/i18n/lang.ts is being sorted alphabetically.

**Comment by abhilipsasahoo03** [comment]
Hey @p-fernandez i have made the required changes! Kindly take a look!

File: docs/docs/notification-center/react-components.md [hunk 1]
```diff
@@ -238,6 +238,7 @@ The `i18n` prop can accept 2 different types of values
           <li><code>zu</code> (Zulu)</li>
           <li><code>zh</code> (Chinese)</li>
           <li><code>sq</code> (Albanian)</li>
+          <li><code>bs</code> (Bosnian)</li>
```
Comment (p-fernandez) [inline docs/docs/notification-center/react-components.md:241]: This is not alphabetically sorted by the iso language code. Could you please ammend?

Comment (p-fernandez) [reply docs/docs/notification-center/react-components.md:241]: Also if you can help us moving the albanian code to its proper place, we would appreciate it.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

**Comment by abhilipsasahoo03** [comment]
@p-fernandez @djabarovgeorge I've reordered the list. Please take a look!

**Review by p-fernandez — APPROVED** [review]
🌟

**Comment by abhilipsasahoo03** [comment]
> 🌟 

Kindly merge :)

**Pull request merged**